rrmngmnt/storage.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import shlex
+from rrmngmnt.service import Service
+
+logger = logging.getLogger(__name__)
+
+LV_CHANGE_CMD = 'lvchange -a %s %s/%s'
+
+
+class NFSService(Service):
+    """
+    Storage management class to maintain NFS services
+    """
+
+    def mount(self, source, target=None, opts=None):
+        """
+        Mounts source to target mount point
+
+        __author__ = "ratamir"
+        :param source: Full path to source
+        :type source: str
+        :param target: Path to target directory, if it is omitted then
+        temporary folder is created and it is the responsible of the
+        caller to remove it at the end
+        :type target
+        :param opts: List of options like: ['-t', 'nfs', '-o', 'vers=3']
+        :type opts: list
+        :return: Path to mount point if succeeded, None otherwise
+        :rtype: str
+        """
+        target = '/tmp/mnt_point' if target is None else target
+        cmd = ['mkdir', target]
+        self.logger.info(
+            "Creating directory %s to use as a mount point", target
+        )
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
+                "Failed to create directory as a mount point for. "
+                "Output: %s, Error: %s ",
+                out, err
+            )
+            return None
+
+        cmd = ['mount', source, target]
+        if opts:
+            cmd.extend(opts)
+        rc, out, err = self.host.run_command(cmd)
+        if rc:
+            self.logger.error(
+                "Failed to mount source %s to target %s. Output: %s",
+                source, target, out
+            )
+            return None
+        return target
+
+    def umount(self, mount_point, force=True, remove_mount_point=True):
+        """
+        Performs 'umount' on 'mount_point' directory and remove
+        the 'mount_point'
+
+        __author__ = "ratamir"
+        :param mount_point: Path to directory that should be un mounted
+        :type mount_point: str
+        :param force: True if force an unmount is required (in case of an
```

Comment:
True if the mount point should be forcefully removed (such as in the case of an unreachable NFS server)